components/script/dom/paintworkletglobalscope.rs

```diff
@@ -446,10 +445,10 @@ impl PaintWorkletGlobalScope {
                     .unwrap_or(10u64);
 
                 select! {
-                    recv(base_channel::after(Duration::from_millis(timeout))) => {
+                    recv(crossbeam_channel::after(Duration::from_millis(timeout))) -> _ => {
```

Comment:
Let's replace this `select!` with:

```rust
receiver.recv_timeout(timeout_duration)
    .map_err(|e| PaintWorkletError::from(e))
```

And then restore this impl, which was removed in the PR that migrated from mpsc to `crossbeam-channel` v0.2:
https://github.com/gterzian/servo/blob/e784f5a9f7a4c3eec92b946b0debd4cbcb942ae7/components/script_traits/lib.rs#L807-L811